File: agent-scan-demo/mobile/flutter/lib/assistant.dart
import 'package:dart_openai/dart_openai.dart';

Future<String> draftReply(String ticket) async {
  final chat = await OpenAI.instance.chat.create(model: 'gpt-4o', messages: []);
  return chat.choices.first.message.content.toString();
}

File: safety-demo/new/web/src/assistant.ts
import OpenAI from 'openai';

const client = new OpenAI();

export async function suggestReply(ticket: string): Promise<string> {
  const r = await client.chat.completions.create({ model: 'gpt-4o', messages: [{ role: 'user', content: ticket }] });
  return r.choices[0].message.content ?? '';
}
